.github/workflows/ci.yml

```diff
@@ -135,7 +135,9 @@ jobs:
         env:
           SECRET_HEADER: ${{ secrets.SECRET_HEADER }}
         run: |
-          ACCOUNT_INDEX=$(curl -X POST -H "Content-Type: text/plain" -H "${SECRET_HEADER}" -d '1' https://fkk3xlyad5tobhmezx4onpvele0zyehu.lambda-url.us-east-2.on.aws/ 2>/dev/null | jq '.[0]' || exit 1)
+          RESPONSE=$(curl -X POST -H "Content-Type: text/plain" -H "${SECRET_HEADER}" -d '1' https://fkk3xlyad5tobhmezx4onpvele0zyehu.lambda-url.us-east-2.on.aws/ 2>/dev/null)
+          echo $RESPONSE
```

Comment:
if the `curl` failed, would still be saved in `RESPONSE`? I don't remember if it will fail and not continue if we do not add some silent fail property

Reply:
Well, `2>/dev/null` is a silent fail mechanism. This silences errors that `curl` considers errors e.g timeouts, wrong URL etc.

In the event of a lambda-specific error, like when we pass a wrong secret, the lambda returns "Unauthorized", with a `401` code. As far as `curl` is concerned, this is not a failure. This is not silenced by `2>/dev/null` and it also doesn't break the CI (although CI will break when `jq` tries to read `.[0]` property from "Unauthorized" and exits with `1` code)

<img width="1408" alt="Screen Shot 2022-08-29 at 16 52 00" src="https://user-images.githubusercontent.com/38781512/187253731-01e62bb6-5e63-4f5b-b89e-83fef639de3f.png">